### DISCUSSION TITLE
Running Multiple Instances

### DISCUSSION THREAD

**Discussion by mcitconsulting** [mikeberger/borg_calendar#218] [General]
Hello 

I am using Borg on a server that has 5 users who log in via remote desktop. Each user has a different username.
Is there a way that I can

A. Run more than 1 instance of Borg at a time - currently each user must close the software before the next can open it.
B. Share the same database across all users - I would like to have a single calendar that all staff can use

Thanks in advance.
Matthew

**Comment by mikeberger** [comment]
A. No
B. I would not recommend this. It is possible to use a cloud service to share the DB (google, dropbox, one drive). I do this to share the db between my own machines. It is 100% inevitable that data will be lost doing this eventually.

BORG used to have 2 different multi-user implementations, but both were removed over 15 years ago. Once people had smartphones and outlook or google calendar, interest in BORG as a multi-user calendar disappeared and maintaining the multi-user code made no sense.

Mike

**Reply by mcitconsulting** [reply]
Thanks Mike. Appreciate the advice. I will leave it as a single user system for the moment then.
